docs/sphinx/reference-scenes.rst

```diff
@@ -566,12 +596,40 @@ Scene Item Functions
 
    :return: The transition for showing or hiding a scene item. *NULL* if no transition is set.
 
+.. deprecated:: 27.2.4
+   Use :c:func:`obs_sceneitem_get_transition()` instead.
+
+---------------------
+
+.. function:: void obs_sceneitem_set_transition_duration(obs_sceneitem_t *item, bool show, uint32_t duration_ms)
+
+   Sets the transition duration for showing or hiding a scene item.
+
+   :param item:        The target scene item
+   :param show:        If *true*, this will set the duration of the show transition;
+                       hide transition otherwise
```

Comment:
```suggestion
   :param show:        If *true*, this will set the duration of the show transition.
                       If *false*, this will set the duration of the hide transition.
```